src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Circle circle = new Circle();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(5) + 1;
+        switch (figureNumber) {
+            case 1:
+                return new Square().createRandomSquare();
+            case 2:
+                return new Circle().createRandomCircle();
+            case 3:
+                return new Rectangle().getRandomRectangle();
+            case 4:
+                return new RightTriangle().getRandomRightTriangle();
+            case 5:
+                return new IsoscelesTrapezoid().getRandomIsoscelesTrapezoid();
+            default:
+                return null;
+        }
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(100 * Math.PI, Color.WHITE.name(), 10);

```

Comment:
Pay attention to [this](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#%D1%81reating-a-figure-dont-pass-expressions-in-constructor) and [this](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants)